docs/datasets.md

```diff
@@ -0,0 +1,78 @@
+# Datasets
+
+**TL;DR:** Every dataset represents a collection of indexed files. Datasets can be
+tagged and merged. Avoid too small datasets (<1000 files) or too many datasets (>100).
+
+Internally all files are indexed by ursadb and stored in datasets.
```

Comment:
`stored in datasets.` Are indexed files stored in datasets?

Reply:
good call, reworded a bit